contribs/src/test/java/com/netflix/conductor/contribs/tasks/http/HttpTaskTest.java

```diff
@@ -68,12 +67,13 @@
     private static final ObjectMapper objectMapper = new ObjectMapper();
     private static String JSON_RESPONSE;
 
-    @ClassRule
-    public static MockServerContainer mockServer = new MockServerContainer(
-            DockerImageName.parse("mockserver/mockserver"));
+    public static MockServerContainer mockServer;
 
     @BeforeClass
     public static void init() throws Exception {
+        mockServer = new MockServerContainer(DockerImageName.parse("jamesdbloom/mockserver:mockserver-5.11.2"));
```

Comment:
If this change does not solve the OOM error, it should be reverted. The `@ClassRule` is less code and more readable.

Reply:
unfortunately, it does not. Reverted